### DISCUSSION TITLE
Can't add photos to album - Documentation outdated?

### DISCUSSION THREAD

**Discussion by embersas** [photoprism/photoprism#2659] [General]
https://docs.photoprism.app/user-guide/organize/albums/

I've created the album, edited and saved ok. My issue comes when trying to add photos to the album.
First step says to select photos (photos doesn't appear in menu, replaced with "search" - minor inconsistency).
Step two says to click the context menu....but I don't have the context menu as shown in the screenshot??

FWIW I'm running docker on Windows, and Firefox as my browser

**Comment by lastzero** [comment]
Screenshots require updating. We welcome contributions. Did you click on the menu bottom right after selecting pictures? Does it work on our demo?

**Comment by graciousgrey** [comment]
Make sure to select photos. Otherwise the context menu won't be displayed 

https://docs.photoprism.app/user-guide/navigate/#7-context-menu
